internal/pkg/secureboot/tpm2/pcr.go

```diff
@@ -79,6 +78,20 @@ func PCRExtend(pcr int, data []byte) error {
 
 	defer t.Close() //nolint:errcheck
 
+	// now we need to check if the TPM is a 2.0 device
+	// we can do this by checking the manufacturer,
+	// if it fails, we can skip the PCR extension
```

Comment:
this is the actual fix, other are just cosmetic changes and for allowing to test the scenario in CI